index.html

```diff
@@ -6,14 +6,17 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./css/main.css" />
     <link rel="icon" href="./favicon.ico" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
     <title>Aziz Mejri - Portfolio</title>
+    <script src="script.js" defer></script>
   </head>
   <body>
     <header>
       <h1 id="logo">Aziz Mejri</h1>
       <button type="button" class="menu-bars">
         <i class="fas fa-bars"></i>
       </button>
+      <i class="fa-solid fa-xmark" id="close"></i>
```

Comment:
![menu](https://user-images.githubusercontent.com/49184579/175476290-24bd918f-3602-4713-b6ea-a514f1423867.png)


- The Hamburger menu option that you get after clicking the hamburger button is not implemented.